operator/api/v1alpha1/keptnapp_types.go

```diff
@@ -60,3 +68,7 @@ type KeptnAppList struct {
 func init() {
 	SchemeBuilder.Register(&KeptnApp{}, &KeptnAppList{})
 }
+
+func (w KeptnApp) GetAppVersionName() string {
```

Comment:
should we ensure the length of the name does not exceed the limit of 64 chars?

Reply:
We always either fail or generate the name as less than 25 char so there is no way here to get a generated app>64